test/TPM_Didymos.jl

```diff
@@ -51,8 +51,10 @@
 
     et_begin = SPICE.utc2et("2027-02-18T00:00:00")  # Start time of TPM
     et_end   = et_begin + 2P₂                       # End time of TPM
-    step     = P₂ / 72                              # Time step of TPM
-    et_range = et_begin : step : et_end
+
+    nsteps_in_period = 72  # Number of steps in one rotation period
+    Δt = P₂ / nsteps_in_period  # Time step of TPM
+    et_range = et_begin : Δt : et_end
```

Comment:
These lines can be simplified. Note that the variable `Δt` is not referenced in other places.
```suggestion
    et_range = range(et_begin, et_end, length=nsteps_in_period+1)
```